### ISSUE TITLE
Need Testing: Frigate Support

### ISSUE THREAD

**Issue by tteck** [tteck/Proxmox#22]
I have created a script that _should_ allow full support to run Frigate in a Home Assistant Container LXC install.

**To add a Home Assistant Container LXC**
In the Proxmox web shell run the following
```yaml
bash -c "$(wget -qLO - https://raw.githubusercontent.com/tteck/Proxmox/main/ct/ha_container.sh)"
```
**To add Frigate support**
In the Proxmox web shell run the following (replace 106 with your Home Assistant Container LXC ID)
```yaml
bash -c "$(wget -qLO - https://raw.githubusercontent.com/tteck/Proxmox/main/misc/frigate-conf.sh)" -s 106
```
[Then configure/add the Frigate Docker image using Docker run in the LXC console.](https://docs.frigate.video/installation)

Please report any findings.

I know nothing about Frigate as I use a POE hardware NVR or I would test myself.

**Comment by B08Z** [comment]
I will test this later today.

**Comment by tteck** [comment]
I've created a script to install a [Vaultwarden](https://github.com/dani-garcia/vaultwarden) LXC. It builds from source, which takes time and resources. After the installation, resources can be set to "**Memory 512 MiB**" and "**Cores 1**" For now, I've left most of the installation process viewable since the rust install needs (for now) user input (requires a "enter key" push), and the lengthy time to complete the full installation.  
```yaml
bash -c "$(wget -qLO - https://raw.githubusercontent.com/tteck/Proxmox/dev/ct/vault_container.sh)"
```
Be Patient, let the script do it's work. Hopefully, you'll eventually see "Successfully created a Vaultwarden LXC Container"
**Please leave feedback, it helps to ensure a stable release.**

![Screenshot 2022-02-04 6 02 22 AM](https://user-images.githubusercontent.com/21342670/152518473-dc17b803-f427-4fa7-a6f7-5e604764c57a.png)

**Comment by B08Z** [comment]
Up and running, very smooth. 

On creation of an account Bitwarden states that the conenction must be HTTPS which is was unable to establish. 

Not done any 'Googling' yet

**Comment by tteck** [comment]
Yes, it needs to run behind a proxy, like Nginx Proxy Manager. I use duckdns.org and just add a new domain (_example:_ _vaultwarden.duckdns.org_) then add it in NPM, with a newly created Let's Encrypt SSL Certificate

**Comment by B08Z** [comment]
Argh, ok. Ill run it behind Traefik. That can handle all the Lets Encrypt. 

Thanks
Awesome work

**Comment by tteck** [comment]
<p align="center"><img src="https://raw.githubusercontent.com/tteck/Proxmox/dev/misc/images/gamuntu.png" alt="GamUntu" </p>

<h1 align="center" id="heading"> GamUntu LXC Container </h1>

<details>
<summary markdown="span">GamUntu LXC</summary>
 
GamUntu is Ubuntu 21.10, Docker, ZFS Support and Hardware Acceleration all baked in! Perfect for Gaming.

To create a new Proxmox GamUntuLXC Container, run the following in the Proxmox web shell.

```yaml
bash -c "$(wget -qLO - https://raw.githubusercontent.com/tteck/Proxmox/dev/ct/gamuntu_container.sh)"
```
<h3 align="center" id="heading">⚡ Default Settings:  2GB RAM - 8GB Storage - 2vCPU ⚡</h3>
 
After the script completes, If you're dissatisfied with the default settings, click on the LXC, then on the **_Resources_** tab and change the **_Memory_** and **_Cores_** settings to what you desire. Changes are immediate.

____________________________________________________________________________________________ 

</details>

**Comment by tteck** [comment]
<p align="center"><img src="https://jellyfin.org/images/banner-dark.svg" height="80"/></p>
<h1 align="center" id="heading"> Jellyfin Media Server LXC </h1>

<details>
<summary markdown="span">Jellyfin Media Server LXC</summary>

<h3 align="center" id="heading"> With Hardware Acceleration Support </h3> 
To create a new Proxmox Jellyfin Media Server LXC, run the following in the Proxmox web shell.

```yaml
bash -c "$(wget -qLO - https://raw.githubusercontent.com/tteck/Proxmox/main/ct/jellyfin_container.sh)"
```
<h3 align="center" id="heading">⚡ Default Settings:  2GB RAM - 8GB Storage - 2vCPU ⚡</h3>

After the script completes, If you're dissatisfied with the default settings, click on the LXC, then on the **_Resources_** tab and change the **_Memory_**, **_Cores_** and **_Root Disk_** (Resize disk) settings to what you desire. Changes are immediate.

**Jellyfin Media Server Interface - IP:8096**

____________________________________________________________________________________________ 

</details>

**Different install method due to issues. So back to testing.**
